digiwf-engine/digiwf-engine-service/src/main/java/de/muenchen/oss/digiwf/cleanup/rest/CleanupInstancesAdminRestController.java

```diff
@@ -0,0 +1,73 @@
+package de.muenchen.oss.digiwf.cleanup.rest;
+
+import de.muenchen.oss.digiwf.cleanup.services.CleanupProcessDefinitionService;
+import jakarta.annotation.security.RolesAllowed;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RequiredArgsConstructor
+@RestController
+public class CleanupInstancesAdminRestController {
+
+    public static final String CLEANUP_ROLE = "admin";
```

Comment:
@simonhir Ich glaub das müssen wir nochmal gerade ziehen. Der Vorschlag zu `admin` kam ja von dir. Macht Sinn, da eine Rolle zu allen Admin-Funktionen zu haben, aber dann gehört eig. auch `clientrole_task_importer` dazu

Reply:
Meiner Meinung nach sollten die ganzen Rollen eh zentral als Const definiert sein und dann kann man die da alle auf Admin setzen oder so.

Gibt theoretisch ja schon folgende Klasse https://github.com/it-at-m/digiwf-core/blob/dev/digiwf-engine/digiwf-engine-service/src/main/java/de/muenchen/oss/digiwf/shared/security/AuthoritiesEnum.java

Reply:
Ja, die hab ich auch schon gesehen. Die andere Rolle befindet sich aber in einem anderen Modul.

Eigentlich ist für so etwas der Unterschied zwischen Rolle und Recht gedacht. Aber ich glaube, dass wir das gleiche Verständnis haben und ich schaue mal, wie ich das etablieren kann

Reply:
Hab das angepasst überall auf `admin`

Das AuthoritiesEnum passt nicht, da dies Rechte beinhaltet und nicht Rollen. Das könnten wir nochmal gerade ziehen, aber ist hier out-of-scope